core/modules/system/system.admin.inc

```diff
@@ -105,10 +105,7 @@ function system_admin_index() {
  */
 function system_themes_page() {
   // Inform the user if theme_debug is enabled.
-  $theme_debug_enabled = config_get('system.core', 'theme_debug');
-  if ($theme_debug_enabled) {
-    backdrop_set_message(t('Theme debugging is enabled. See the <a href="!url">Status report</a> page for more info.', array('!url' => url('admin/reports/status'))), 'warning', FALSE);
-  }
+  theme_debug_enabled_warning();
```

Comment:
Any function that starts with `theme_` you would expect to be override-able by the theme system, but this is not. For clarity, I recommend we rename this to `system_theme_debug_enabled_warning()`, unless this actually is intended to be overridden by a theme, in which case we need `system_theme()` to be updated and call this using `theme()` like other theme functions and templates.

Reply:
I think that `system_theme_debug_enabled_warning()` as you suggest is the right way to go @quicksketch for all the reasons. One question though: I have placed the actial function in theme.inc (because it was called `theme_`something), but I feel that it should now be moved elsewhere. Should it be in `common.inc`, or perhaps `errors.inc`, or `utility.inc`? Elsewhere?

Reply:
`theme.inc` is always loaded ?

Reply:
..moved the function in system.module